common/source/docs/common-downloads_advanced_user_tools.rst

```diff
@@ -421,22 +392,6 @@ DATE POSTED	April 25, 2013
 Arduino USB driver for the Atmega32u2 USB interface chip on APM 2
 
 
```

Comment:
I wasn't able to find anything and it referenced the HIL simulator which i thought wasn't supported anymore. Is this file needed for general flightgear setup w/ SITL?